spec/openapi.yaml

```diff
@@ -426,6 +426,44 @@ paths:
         503:
           $ref: "#/components/responses/ServiceUnavailable"
 
+  /api/v1/update/{env}:
+    parameters:
+      - in: path
+        name: storage
+        description: The target environment (python or system) for update
+        required: true
+        schema:
+          type: string
+          enum: ["python", "system"]
```

Comment:
for now, use only one type: `python`. When we will implement system and or firmware, we will add it.